DockerComposeExamples/RunMultipleFlinkJobs/docker-compose.yaml

```diff
@@ -0,0 +1,177 @@
+---
+services:
+  # Copied from JsonToAvro project
+  # Start of JsonToAvro service
+  jsontoavro-test:
```

Comment:
would it make sense to have a component test that uses both jobs?

Reply:
I thought about it, but this is more an example on what is needed when copying docker compose from existing project and wanting to run multiple together. 

But I could also see a case for developers not knowing you can use our component test framework to write a test for the entire pipeline.